### DISCUSSION TITLE
Feedback for Application Process

### DISCUSSION THREAD

**Discussion by fenix1stroke** [community/community#144211] [GitHub Education]
### Hi everyone,

Hey there,

can we please talk about why the only accepted file formats are image formats? I do not own a printer and my students license has the crucial information distributed equally on the front and back of the card. Why can't I in this day and age just upload the generally accepted confirmation of enrollment that has a verification link and qr-code on the document as PDF? That should make the process easier not only for me as an applicant but also for the people who review the applications. I had to take a screenshot, save it and then upload that screenshot instead of just dropping the PDF file in there. I know that doesn't sound like a big hassle but I can imagine people really struggling with how it is right now.

Anyway have a great day, thanks for your attention.

**Comment by gr1shan1a** [comment]
it’d be way easier if we could just upload a PDF version of the enrollment confirmation. That document usually has a verification link and a QR code, which would make things simpler for both applicants and the reviewers. Instead, I had to take a screenshot, save it as an image, and then upload that. It’s an extra step that could be confusing or annoying for others too.
